BusinessLogicLayer/Initializer/Implementation/Initialize.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using BusinessLogicLayer.Initializer.Interfaces;
+using BusinessLogicLayer.ViewModels;
+using DataAccessLayer.DataBaseModels;
+using Microsoft.EntityFrameworkCore.Metadata;
+using OfficeOpenXml;
+
+namespace BusinessLogicLayer.Initializer.Implementation
+{
+    public class Initialize : IIInitialize
+    {
+        private UnitOfWork _unitOfWork = new UnitOfWork();
+
+        public string CreatePath(string directoryName = "", string fileName = "Languages")
+        {
+            string rootPath = Directory.GetCurrentDirectory() + @"\wwwroot\Dictionary\" + directoryName + fileName + ".xlsx";
+            Console.WriteLine(rootPath);
+            return rootPath;
+        }
+
+        public ExcelWorksheet GetDataFromFile(string path)
+        {
+            FileInfo fileInfo = new FileInfo(path);
+            ExcelPackage.LicenseContext = LicenseContext.NonCommercial;
+            ExcelPackage package = new ExcelPackage(fileInfo);
+            ExcelWorksheet worksheet = package.Workbook.Worksheets.FirstOrDefault();
+            return worksheet;
+        }
+
+        public List<ParsedData> ParseData(ExcelWorksheet worksheet)
+        {
+            var parsedData = new List<ParsedData>();
+
+            if (worksheet != null)
+            {
+                // get number of rows and columns in the sheet
+                int rows = worksheet.Dimension.Rows;
+                int columns = worksheet.Dimension.Columns;
+
+                // loop through the worksheet rows and columns
+                for (int i = 2; i <= rows; i++)
```

Comment:
Please always add explaining comments when you use hard-coded strings/numbers. E.g. here it's not clear why do you start from 2, and not from 0.